R/release.R

```diff
@@ -140,6 +166,13 @@ release_extra <- function(env = NULL) {
   }
 }
 
+safe_pkg_env <- function() {
+  tryCatch(
+    pkg_env(project_name()),
```

Comment:
```suggestion
    ns_env(project_name()),
```

In case package wasn't loaded-all with internal exports?